app/src/main/res/values-hdpi/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="feed_info_background_image_height">198dp</dimen>
```

Comment:
The problem with ldpi/hdpi is that it only says something about the screen resolution, not the actual size. There might be large screens with low resolution or small screens with high resolution. I don't know a perfect solution but maybe you can play around with [some other qualifiers (Table 2)](https://developer.android.com/guide/topics/resources/providing-resources)

Most old+small devices have a low resolution but I would prefer to use a qualifier that is actually about the size.